generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.component.html

```diff
@@ -31,7 +31,7 @@
                     <!-- jhipster-needle-add-entity-to-menu - JHipster will add entities to the menu here -->
                 </ul>
             </li>
-            <li *ngSwitchCase="true" <%=jhiPrefix%>HasAuthority="ROLE_ADMIN" ngbDropdown class="nav-item dropdown pointer">
```

Comment:
You cannot have 2 structural diirectives on same element and in this case the *ngStwitch="true" means user is authenticated which is already covered by HasAnyAuthority.